web/docs/setup-guides/deploy-a-node/README.md

```diff
@@ -201,3 +142,10 @@ There exist two ways stop a server:
    `pkill -2 tenzir-node`.
 
 Sending the process a SIGTERM is the same as hitting CTRL+C.
+
+## Automate the deployment
+
+Read our guides on automating the deployment of a node:
+
+- [Systemd](systemd.md)
+- [Ansible](ansible.md)
```

Comment:
The wording seems a bit off, "automating the deployment" is strictly the realm of tools like Ansible etc., not systemd.

Reply:
Yeah, it's a bit odd indeed. Any suggestion on how to integrate the systemd page?

Reply:
I spoke with @tobim about this. It's not technically correct but we'll leave it like this for now.